### PR TITLE
test: Fix local storage test 

### PR DESCRIPTION
**Pull request by juanpicado**
Add more checks to test for local-storage

* addVersion
* updateVersions

### CODE REVIEW

File: test/unit/api/local-storage.spec.js [hunk 1]
```diff
@@ -22,6 +27,55 @@ describe('LocalStorage', () => {
...
+        resolve(data);
+      });
+    });
+  };
+  const addTarballToStore = (pkgName: string, tarballName) => {
+    return new Promise((resolve, reject) => {
+      const tarballData = JSON.parse(readMetadata('addTarball'));
+      const stream = storage.addTarball(pkgName, tarballName);
+
+      stream.on('error', function(err) {
```
Comment (ayusharma) [inline test/unit/api/local-storage.spec.js]: Arrow functions

File: test/unit/api/local-storage.spec.js [hunk 2]
```diff
@@ -22,6 +27,55 @@ describe('LocalStorage', () => {
...
+        expect(err).toBeNull();
+        reject();
+      });
+      stream.on('success', function() {
```
Comment (ayusharma) [inline test/unit/api/local-storage.spec.js]: here also,

File: test/unit/api/local-storage.spec.js [hunk 3]
```diff
@@ -71,38 +125,125 @@ describe('LocalStorage', () => {
...
+          expect(metadata[DIST_TAGS]['beta']).toBe('3.0.0');
+          expect(metadata[DIST_TAGS]['latest']).toBe('2.0.0');
+          done();
+        });
+      });
+
+      test('should fails mergeTags version not found', async (done) => {
+        const pkgName = 'merge-tags-test-1';
+        await addPackageToStore(pkgName, generatePackageTemplate(pkgName));
+        // const tarballName: string = `${pkgName}-${version}.tgz`;
```
Comment (ayusharma) [inline test/unit/api/local-storage.spec.js]: Remove this

Comment (juanpicado) [reply test/unit/api/local-storage.spec.js]: done :)

**Pull request merged**